types/react-autosuggest/index.d.ts

```diff
@@ -93,8 +93,9 @@ declare namespace Autosuggest {
         containerProps: {
             id: string;
             key: string;
+            className: string;
             ref: any;
-            style: any;
+            role?: string;
```

Comment:
I don't think `role` should be optional based on this (which is what is ultimately calling `renderSuggestionsContainer`
https://github.com/moroshko/react-autowhatever/blob/a74c882d4f63fabe7856e5d90b93c2af81c89a5b/src/Autowhatever.js#L342